src/panel-triggers/components/AlertList/AlertList.tsx

```diff
@@ -97,15 +103,16 @@ export default class AlertList extends PureComponent<AlertListProps, AlertListSt
     return this.props.onProblemAck(problem, data);
   }
 
-  getFilteredProblems = (textFilter: string, priorityFilter: number, categoryFilter: string, maintenanceFilter: boolean, sortOption: string) => {
+  getFilteredProblems = (textFilter: string, priorityFilter: number, categoryFilter: string, maintenanceFilter: boolean, testTagFilter: boolean, sortOption: string) => {
     const textFilterLowerCase = textFilter.toLowerCase();
     let filteredProblems = this.props.problems.filter((problem: ProblemDTO) => {
       return (
         (problem.comments.toLowerCase().indexOf(textFilterLowerCase) > -1 ||
         problem.description.toLowerCase().indexOf(textFilterLowerCase) > -1) &&
         (priorityFilter === -1 || problem.severity === priorityFilter.toString()) &&
         (categoryFilter === 'all' || problem.opdata === categoryFilter) &&
-        (!maintenanceFilter || (problem.hosts.length > 0 && problem.hosts[0].maintenance_status === '0'))
+        (!maintenanceFilter || (problem.hosts.length > 0 && problem.hosts[0].maintenance_status === '0')) &&
+        (!testTagFilter || problem.tags.length === 0 || problem.tags.some((e) => e.tag !== 'test'))
```

Comment:
I think this is currently the wrong way up. Shouldn't it be `!problem.tags.some((e) => e.tag === 'test'))` ?
So that the problem is filtered away if there is any tag with key `test` ?

Reply:
In commit 04a2e1f925826ca91666d580df8033ea79d1ebdf.